client/src/assets/images/bc-logo-mini.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="40px" height="28px" viewBox="0 0 40 28" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Logo</title>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="2.0.1.0---Confirmation---Passed-All----Mobile" transform="translate(-56.000000, -16.000000)" fill-rule="nonzero">
+            <g id="Header_Traveller_Support" transform="translate(56.000000, 10.000000)">
```

Comment:
Can we change the id here? Instead of "Header_Traveller_Support" maybe "Header_for_Form_Mini" or something similar to match the name in the other SVG? You can change this without affecting the image :)